apps/web-giddh/src/app/daybook/daybook.component.html

```diff
@@ -167,138 +170,340 @@
+                            <mat-cell *matCellDef="let entry">
+                                <span class="responsive-label" [hidden]="!daybookData?.entries?.length">{{
+                                    commonLocaleData?.app_rate
+                                }}</span>
+                            </mat-cell>
+                        </ng-container>
+                        <ng-container *ngIf="isEntryExpanded" matColumnDef="hsn_sac">
+                            <mat-header-cell [hidden]="!daybookData?.entries?.length" *matHeaderCellDef>{{
+                                commonLocaleData?.app_hsn_sac
+                            }}</mat-header-cell>
+                            <mat-cell *matCellDef="let entry">
+                                <span class="responsive-label" [hidden]="!daybookData?.entries?.length">{{
+                                    commonLocaleData?.app_hsn_sac
+                                }}</span>
+                            </mat-cell>
+                        </ng-container>
+                        <ng-container *ngIf="isEntryExpanded" matColumnDef="sku">
+                            <mat-header-cell [hidden]="!daybookData?.entries?.length" *matHeaderCellDef>{{
+                                commonLocaleData?.app_sku
+                            }}</mat-header-cell>
+                            <mat-cell *matCellDef="let entry">
+                                <span class="responsive-label" [hidden]="!daybookData?.entries?.length">{{
+                                    commonLocaleData?.app_sku
+                                }}</span>
+                            </mat-cell>
+                        </ng-container>
+                        <ng-container *ngIf="isEntryExpanded" matColumnDef="warehouse">
+                            <mat-header-cell [hidden]="!daybookData?.entries?.length" *matHeaderCellDef>{{
+                                commonLocaleData?.app_warehouse
+                            }}</mat-header-cell>
+                            <mat-cell *matCellDef="let entry">
+                                <span class="responsive-label" [hidden]="!daybookData?.entries?.length">{{
+                                    commonLocaleData?.app_warehouse
+                                }}</span>
+                            </mat-cell>
+                        </ng-container>
 
-                        <tbody
-                            *ngFor="let entry of daybookData?.entries"
-                            (click)="expandEntry(entry)"
-                            class="daybook-table-tbody"
-                        >
-                            <tr
-                                (dblclick)="showUpdateLedgerModal(entry)"
-                                (keydown.enter)="showUpdateLedgerModal(entry)"
+                        <ng-container matColumnDef="expandedDetail">
+                            <mat-cell
+                                class="pd-0 w-100"
```

Comment:
@ravinderwalkover use class p-0, in place of pd-0 for padding 0 @ravinderwalkover

Reply:
Done